### DISCUSSION TITLE
Ability to identify which devices are not responding

### DISCUSSION THREAD

**Discussion by CalleAnka** [frenck/spook#825] [Ideas]
Home Assistant is stupid because it not tell if some device is not sen it state anymore. Example; I have temperature sencors that not shown in dashboard. Instead their values are trackking to database. Sometimes their battery ends but I can't know it. 

Excellent property to Spook could check devices last sent values and if update was too sparse, Spook could inform me from it.

**Comment by frenck** [comment]
That is not a task of any integration. If want want to be notified of something happening: create an automation. That is what it is for 🤷‍♂️
